src/blst-native/secretKey.ts

```diff
@@ -1,5 +1,6 @@
 import * as blst from "@chainsafe/blst";
-import {bytesToHex, hexToBytes, isZeroUint8Array, randomBytes} from "../helpers/index.js";
+import crypto from "crypto";
```

Comment:
This will help in tree-shaking for the final build. 

```suggestion
import {randomBytes} from "crypto";
```

Reply:
This is not browser code, how does it matter for tree shaking here? destructuring esm modules makes no difference afaik on the server

Reply:
Someone can use this package somewhere which endup for browser support.

Reply:
As it's only for `bls-native` which is not gonna be used to build web distribution so we are good here.